### PR TITLE
Feature/new 2d models

### PR DESCRIPTION
**Pull request by mosheman5**
Add two new models, using pretrained versions that should improve performance on small datasets.
also add a preprocessor file that includes the melspectrogram representation, that should provide better results than Spectrogram (at least for some call types)

### CODE REVIEW

File: soundbay/conf/preprocessors/_mel_preprocessors.yaml
Comment (shaing10) [inline soundbay/conf/preprocessors/_mel_preprocessors.yaml:1]: win_length = n_fft by definition - is that what we want?

BTW - https://github.com/deep-voice/soundbay/blob/brazil-slowfast/soundbay/conf/preprocessors/_preprocessors_mel_unit.yaml

File: soundbay/models.py [hunk 1]
```diff
@@ -320,3 +320,47 @@ def forward(self, x):
...
+
+
+class Squeezenet2D(nn.Module):
+    def __init__(self, num_classes=2, pretrained=True):
+        super(Squeezenet2D, self).__init__()
+
+        self.squeezenet = torch.hub.load('pytorch/vision:v0.10.0', 'squeezenet1_0',
+                                         pretrained=pretrained)
+        # number of features from existing squeezenet
+        num_features = 1000
```
Comment (shaing10) [inline soundbay/models.py:332]: if it is possible, maybe it is better to read it from `self.squeezenet` / from the config

**Review by shaing10 — APPROVED** [review]
2 small nice-to-have suggestions

File: soundbay/conf/preprocessors/_mel_preprocessors.yaml
Comment (mosheman5) [reply soundbay/conf/preprocessors/_mel_preprocessors.yaml:1]: https://pytorch.org/audio/main/generated/torchaudio.transforms.MelSpectrogram.html
It is the defaults behavior of melspecs, so I think it's gonna be fine.
regarding the BTW - lol. Any reason we didn't merge to master?

**Pull request merged**